osu.Game/Overlays/BeatmapSet/BeatmapRulesetTabItem.cs

```diff
@@ -3,143 +3,87 @@
-                    Children = new Drawable[]
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = colourProvider.Background6
+                    },
+                    count = new OsuSpriteText
                     {
-                        name = new OsuSpriteText
-                        {
-                            Anchor = Anchor.Centre,
-                            Origin = Anchor.Centre,
-                            Text = value.Name,
-                            Font = OsuFont.Default.With(size: 18),
-                        },
-                        new Container
-                        {
-                            Anchor = Anchor.Centre,
-                            Origin = Anchor.Centre,
-                            AutoSizeAxes = Axes.Both,
-                            Masking = true,
-                            CornerRadius = 4f,
-                            Children = new Drawable[]
-                            {
-                                new Box
-                                {
-                                    RelativeSizeAxes = Axes.Both,
-                                    Colour = Color4.Black.Opacity(0.5f),
-                                },
-                                count = new OsuSpriteText
-                                {
-                                    Alpha = 0,
-                                    Anchor = Anchor.Centre,
-                                    Origin = Anchor.Centre,
-                                    Margin = new MarginPadding { Horizontal = 5f },
-                                    Font = OsuFont.Default.With(weight: FontWeight.SemiBold),
-                                }
-                            }
-                        }
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Margin = new MarginPadding { Horizontal = 5f },
+                        Font = OsuFont.Default.With(weight: FontWeight.SemiBold),
+                        Colour = colourProvider.Foreground1,
                     }
-                },
-                bar = new Box
-                {
-                    Anchor = Anchor.BottomCentre,
-                    Origin = Anchor.BottomCentre,
-                    RelativeSizeAxes = Axes.X,
-                },
-                new HoverClickSounds(),
-            };
+                }
+            });
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
 
             BeatmapSet.BindValueChanged(setInfo =>
             {
                 var beatmapsCount = setInfo.NewValue?.Beatmaps.Count(b => b.Ruleset.Equals(Value)) ?? 0;
 
                 count.Text = beatmapsCount.ToString();
-                count.Alpha = beatmapsCount > 0 ? 1f : 0f;
+                countContainer.FadeTo(beatmapsCount > 0 ? 1 : 0);
```

Comment:
Kind of a strange change, as there doesn't seem to be any functional difference between `FadeTo` without explicit `duration` and directly setting `Alpha`.

Reply:
Just easier to read imo, nothing special